apm-sniffer/optional-plugins/mybatis-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/mybatis/MyBatisMethodMatch.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.plugin.mybatis;
+
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+import static net.bytebuddy.matcher.ElementMatchers.named;
+
+public enum MyBatisMethodMatch {
+    INSTANCE;
+
+    public ElementMatcher<MethodDescription> getMyBatisMethodMatcher() {
+        return named("selectOne").or(named("selectList"))
```

Comment:
I saw that you prevent repeated generation of localSpan code in the interceptor. It is because of such code.
![image](https://user-images.githubusercontent.com/10150229/116063768-ed931f00-a6b7-11eb-903b-3d7db87a711d.png)
 right? If it is, the code is ok, but there code `matcher method` can be clearer.

Reply:
Yes, I want to avoid wasting resources by generating unimportant localSpan.
In some scenarios, selectList() is called directly.
![image](https://user-images.githubusercontent.com/28091237/116065272-7068a980-a6b9-11eb-9e97-06384add0ad6.png)

Reply:
So use more API such as `takesArgument` is better

Reply:
I think your suggestion is very good and I will optimize it.

Reply:
Done.
![image](https://user-images.githubusercontent.com/28091237/116073125-93e42200-a6c2-11eb-98b5-c2259fd4ae94.png)

Reply:
Mean, can use `named("selectOne").and(takesArguments(2))` instead of `named("selectOne")`?  what do you think about？

Reply:
I don't recommend this, because sometimes it will enter a parameter's selectOne first.

Reply:
OK